pkg/data/collector.go

```diff
@@ -105,29 +105,42 @@ func (c *Collector) collectBidFromRelay(ctx context.Context, relay *builder.Clie
 
 func (c *Collector) collectFromRelay(ctx context.Context, relay *builder.Client) {
 	logger := c.logger.Sugar()
-
+	
 	relayID := relay.PublicKey
 
 	slots := c.clock.TickSlots(ctx)
+	nextSlot := slots + uint64(1)
```

Comment:
@Ross-Kitsis I get a warning here on VSCode about chan uint64 and uint64, not sure how to fix it

Reply:
Slots is a channel that outputs integers but its no an integer itself. The actual slot is retried on line 119 ```case slot := <-slots:```

You would have to do this calculation there.